Sources/KlaviyoSwift/StateManagement.swift

```diff
@@ -230,6 +233,15 @@ struct KlaviyoReducer: ReducerProtocol {
             state.enqueueRequest(request: request)
             return .none
 
+        case let .setPushEnablement(enablement):
+            guard let pushToken = state.pushTokenData?.pushToken else {
+                return .none
+            }
+
+            return .run { send in
```

Comment:
You might be missing some coverage on this bit. You could add a test that send this directly or just change the state so that you have a push token in one of the tests.

Reply:
should be addressed now, thanks